app/views/submissions/_form.html.erb

```diff
@@ -63,8 +70,61 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12 mt-2" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" class="submission-initially-hidden">
+				<i class="fas fa-check"></i>
+				<%=t('submission.correct_file_size')%>
+			</div>
+			<div class="col-12" id="file-size-too-big" class="submission-initially-hidden">
+				<i class="fas fa-exclamation-triangle"></i>
+				<%=t('submission.file_size_too_big')%>
+			</div>
+			<div class="col-12" id="file-size-way-too-big" class="submission-initially-hidden">
+				<i class="fas fa-bomb"></i>
+				<%=t('submission.file_size_way_too_big')%>
+			</div>
+			<div class="col-12" id="file-optimize" class="submission-initially-hidden">
+				<small>
+					<%= t("submission.optimization_help_html")%>
+					<%= helpdesk(t('submission.optimization_info'),true)%>
+				</small>
```

Comment:
The `<small>` block should IMO be hidden after successful optimization as it refers to a button that does not exist afterwards.

Reply:
Also the button?

Reply:
Hmm, the button contains the size information, so I think it should stay (or you find another place to put the size information, like in parentheses after `#file-size-correct` but I do not know if this is totally consistent with the other cases).